### PR TITLE
Fix animation expression not being visited for use count of property

### PR DESCRIPTION
**Pull request by ogoffart**
resulting in properties being optimized when they shouldn't

### CODE REVIEW

File: internal/compiler/generator/rust.rs [hunk 1]
```diff
@@ -114,6 +114,7 @@ fn rust_primitive_type(ty: &Type) -> Option<proc_macro2::TokenStream> {
 fn rust_property_type(ty: &Type) -> Option<proc_macro2::TokenStream> {
     match ty {
         Type::LogicalLength => Some(quote!(sp::LogicalLength)),
+        Type::Easing => Some(quote!(sp::EasingCurve)),
```
Comment (tronical) [inline internal/compiler/generator/rust.rs:117]: Is that an unrelated change? (I don't mind it being here, just curious)

**Review by tronical — APPROVED** [review]

File: internal/compiler/generator/rust.rs (hunk 1 above)
Comment (ogoffart) [reply internal/compiler/generator/rust.rs:117]: The test add a `property <easing>` which fails.

The gallery also have that. But in the gallery, the property is optimized out so the rust code compile, but not here.
